### ISSUE TITLE
Compilation error for instanceof

### ISSUE THREAD

**Issue by sergeyklay** [zephir-lang/zephir#1828]
Steps to reproduce:

```zep
public function test07(string test) -> bool
{
    var obj;
    let obj = new \stdClass;

    return obj instanceof test;
}
```

The error message:
```
In file included from /home/klay/.phpenv/versions/7.3.0-zts-debug/include/php/Zend/zend.h:27:0,
                 from /home/klay/.phpenv/versions/7.3.0-zts-debug/include/php/main/php.h:33,
                 from /home/klay/workspace/c/zephir-issue-1802/ext/bug/instance.zep.c:6:
/home/klay/workspace/c/zephir-issue-1802/ext/bug/instance.zep.c: In function ‘zim_Bug_Instance_test07’:
/home/klay/.phpenv/versions/7.3.0-zts-debug/include/php/Zend/zend_types.h:649:39: error: invalid type argument of unary ‘*’ (have ‘zval {aka struct _zval_struct}’)
 #define Z_STRVAL_P(zval_p)   Z_STRVAL(*(zval_p))
```

**Comment by joeyhub** [comment]
This has always been a problem in php, you have to use is_a.

**Comment by sergeyklay** [comment]
Fixed in the  #1829
